packages/fast-components-react-msft/src/button/button.props.ts

```diff
@@ -25,6 +25,16 @@ export interface ButtonHandledProps
      * The Button appearance
      */
     appearance?: ButtonAppearance;
+
+    /**
+     * The preceding slot
+     */
+    beforeSlot?: (classname?: string) => React.ReactNode;
```

Comment:
is there a reason not to go with something unrelated to "slots"? I think that may be confusing; what about `beforeContent` an `afterContent`?

Reply:
Yeah, I could see the usage of slots being confusing. I'm ok with `beforeContent` and `afterContent` if everyone agrees.